src/style.css

```diff
@@ -2,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
+  padding-inline-start: 0;
+  list-style-type: none;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Consider using classes for styling instead of tag names. If `.stars__star` is a class for an `li` element, it's better to style it using the class name rather than the tag to follow best practices and maintain specificity.